lib/main.dart

```diff
@@ -51,7 +50,7 @@ class _MyAppState extends State<MyApp> {
   @override
   Widget buildApp(BuildContext context) {
     return _amplifyConfigured
-        ? const HomePage(title: 'Flutter Demo Home Page')
+        ? const HomePage(title: 'Amplify Expense ')
```

Comment:
nit, can extra space at and of string be removed? Also, can remove @override here (my bad bc I wrote this) [ducks].